content/en/includes/ki-aws-image-cache.md

```diff
@@ -0,0 +1,15 @@
+#### AWS image caching
```

Comment:
@aelath can you check if this regression was fixed in 2.23.4 or if it still exists?